draft-beky-httpbis-metadata.md

```diff
@@ -149,16 +161,16 @@ This document defines a new HTTP/2 setting identifier, SETTINGS_ENABLE_METADATA,
 with value 0x4d44.  It also defines a new HTTP/3 setting identifier,
 SETTINGS_ENABLE_METADATA, with value 0x4d44.
 
-An endpoint that supports METADATA frames SHOULD advertise that by sending
+An endpoint that supports METADATA frames _SHOULD_ advertise that by sending
 SETTINGS_ENABLE_METADATA with value 1 on each connection.  A value of 0
 indicates that the endpoint does not support METADATA frames.  A value other
-than 0 or 1 MUST not be sent.  The initial value is 0.  For HTTP/2,
-SETTINGS_ENABLE_METADATA MUST not be sent in any SETTINGS frame other than the
+than 0 or 1 _MUST_ not be sent.  The initial value is 0.  For HTTP/2,
+SETTINGS_ENABLE_METADATA _MUST_ not be sent in any SETTINGS frame other than the
 first one.
 
-An endpoint MAY send METADATA frames before it learns that the peer supports
+An endpoint _MAY_ send METADATA frames before it learns that the peer supports
 them.  For example, a proxy might chose to forward METADATA frames, or it might
-chose to buffer them, before it receives a SETTINGS frame.  An endpoint SHOULD
+chose to buffer them, before it receives a SETTINGS frame.  An endpoint _SHOULD_
```

Comment:
uber-nit: "_SHOULD_ NOT" --> "_SHOULD NOT_"

Reply:
Done.